packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -139,7 +146,6 @@ class SocksProxyConnection {
       dummyServer.emit('connection', this.internal);
       dummyServer.on('secureConnection', internalTLS => {
         debugLogger.log('client-certificates', `Browser->Proxy ${this.host}:${this.port} chooses ALPN ${internalTLS.alpnProtocol}`);
-        internalTLS.on('close', () => this.socksProxy._socksProxy.sendSocketEnd({ uid: this.uid }));
```

Comment:
My assumption is that the 'close' event of the `tls.TLSSocket` will bubble down onto the `internal` `net.Socket`.